.gitignore

```diff
@@ -1,5 +1,6 @@
 # Temporary
 data/*
```

Comment:
shall we drop data from .gitignore then?

Reply:
Dropped